### ISSUE TITLE
Create generator for .fit_generator()

### ISSUE THREAD

**Issue by jamesmf** [jamesmf/charLevelCNN#2]
Given a fixed validation and test set, the training data should be streamed using fit_generator()
